### PR TITLE
Pull request for Bookstore: styling

### PR DESCRIPTION
**Pull request by zuhebahmed88091**
In previous branch->

- [ ] Added Redux Toolkit. 
- [ ] Created a directory that contains all Redux logic.
- [ ] Configured a Redux store. 
- [ ] Defined a slice of state for books that:

    -     Includes an array of books (initial state: empty array)
    -     Includes a reducer that adds a book
    -     Includes a reducer that removes a book

- [ ] Defined a slice of state for categories that:

    -     Includes an array of categories (initial state: empty array)
    -     Includes a reducer that checks the status and always returns "Under construction"
- [ ] Wrapped my <App> with the <Provider> component from react-redux
- [ ] Passed my configured store into the <Provider>
- [ ] Created an initialState variable for your booksSlice
- [ ] Displayed books, received from the slice, in a reusable component
- [ ] Used Dispatch actions using useDispatch
- [ ] Added a <Button> component, which includes:
    - An event handler that adds a book to the books array
- [ ] Added a <Button> component, which includes:
    - An event handler that removes a book from the books array (by id)
- [ ] Made use of the [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) as your backend
- [ ] Used [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) to make your HTTP requests to the backend
- [ ] Fetched books from the Bookstore API after the component has rendered
- [ ] Refactored add book and remove book features to persist changes in the server

In this branch->

- [ ] Designed the Bookstore according to [Zeplin](https://zeplin.io/)
- [ ] Added missing JSX elements to your project 

Thank you for reviewing my PR.

### CODE REVIEW

File: src/components/nav.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-  <nav>
-    <h1>Bookstore CMS</h1>
-    <ul>
-      <li>
-        <NavLink to="/">
+  <nav className="nav-container">
+    <h1 className="nav-heading">Bookstore CMS</h1>
+    <ul className="nav-ul">
+      <li className="nav-li">
+        <NavLink to="/" className="links">
```
Comment (aamirkhan2478) [inline src/components/nav.js]: - [ ] Well done you did a very nice job but there are some little changes so kindly fix it.
- [x] If you check the design the selected link is with this `#121212` and the link which is not selected is with this color `#fafafa` but in you can you just add one color so what you do when the use clicks on the book link the link color should be `#121212` and the other link color should be `#fafafa` same with categories link so kindly fix this issue. Check below the comparison between your design and Zaplin's design

**Your Design**
Screenshot from 2023-07-20 19-11-41

**Zaplin Design**
Screenshot from 2023-07-20 19-11-41

- [x] Also if you check your Zaplin design all letters of link are capital letter so kindly add capital letter as well

File: src/components/book.js [hunk 2]
```diff
@@ -11,14 +14,38 @@ const Book = ({ singleBook }) => {
...
+          <div className="vertical-divider" />
+          <Button className="remove-btn" onClick={() => handleRemove()} label="Remove" />
+          <div className="vertical-divider" />
+          <Button className="edit-btn" label="Edit" />
+        </div>
+      </article>
+      <article className="progressbar-wrap">
+        <div className="circle-bar">
+          <CircularProgressbar value={45} />
+        </div>
```
Comment (aamirkhan2478) [inline src/components/book.js]: - [ ] Please change the color of your circular progress bar the color is not the same as the Zaplin design so kindly change the color of your circular progress bar.

**Your Design**
Screenshot from 2023-07-20 19-51-50

**Zaplin Design**
Screenshot from 2023-07-20 19-10-41

File: src/components/nav.js [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
-        <NavLink to="/categories">
+      <li className="nav-li">
+        <NavLink to="/categories" className="links">
           Categories
         </NavLink>
       </li>
     </ul>
+    <div className="profile-wrap">
+      <CgProfile className="profile-icon" />
+    </div>
```
Comment (aamirkhan2478) [inline src/components/nav.js]: - [x] Please use the same user icon as mentioned in the Zaplin design but your design is not the same as Zaplin's design.

**Your Design**
Screenshot from 2023-07-20 19-12-09

**Zaplin Design**
Screenshot from 2023-07-20 19-12-00

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @zuhebahmed88091 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/book.js (hunk 2 above)
Comment (zuhebahmed88091) [reply src/components/book.js]: I have copied the bar color->#307bbe from zeplin.
Issue resolved

File: src/components/nav.js (hunk 3 above)
Comment (zuhebahmed88091) [reply src/components/nav.js]: I have tried to find the exact icon everywhere in:

- FontAwesome
- Bootstrap
- etc.
- Even in your repo

Unfortunately I could not find this exact icon for **free** anywhere. This one is available in Pro version.
So I have tried my best till now and applied the closest that goes similar to the resemblance to the mentioned icon.

File: src/components/nav.js (hunk 1 above)
Comment (zuhebahmed88091) [reply src/components/nav.js]: I have applied `useState ` and `useLocation` Hooks to resolve this issue.

- [x] Resolved.

**Comment by zuhebahmed88091** [comment]
@aamirkhan2478 Thanks for the review.

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @zuhebahmed88091, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**